spec/support/stories/profile_stories.rb

```diff
@@ -10,7 +10,7 @@ def as_a_registered_user
       account: Fabricate(:account, username: 'bob')
     )
 
-    Web::Setting.where(user: bob).first_or_initialize(user: bob).update!(data: { introductionVersion: 201812160442020 }) if finished_onboarding # rubocop:disable Style/NumericLiterals
+    Web::Setting.where(user: bob).first_or_initialize(user: bob).update!(data: { introductionVersion: 201_812_160_442_020 }) if finished_onboarding
```

Comment:
Should this just be date formatted like the existing pattern exception?
```suggestion
    Web::Setting.where(user: bob).first_or_initialize(user: bob).update!(data: { introductionVersion: 2018_12_16_0442020 }) if finished_onboarding
```

Reply:
Oh, it has one to many digits at the end, so maybe the trailing 0 should be dropped, unless this number is actually correct

Reply:
Dropped the trailing 0, since it doesn't have it where it was defined `export const INTRODUCTION_VERSION = 20181216044202;`

Reply:
Good catch - updated to match constant from JS onboarding class.